src/syncAzure.builds

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Why is this a .builds file? .builds files are about wrapping other projects and not just for general projects, they should always import dir.traversals.targets.